target/linux/realtek/image/Makefile

```diff
@@ -54,7 +54,7 @@ endef
 define Device/d-link_dgs-1210-10p
   $(Device/d-link_dgs-1210)
   DEVICE_MODEL := DGS-1210-10P
-  DEVICE_PACKAGES += lua-rs232
+  DEVICE_PACKAGES += lua-rs232 rtl83xx-poe
```

Comment:
The rtl83xx-poe package isn't in master yet (last time I checked only in blogic's tree) so you can't have it depend on that for now (see e.g. the GS1900-10HP/8HP definitions).

Reply:
You're right it didn't pull in rtl38xx-poe.  I didn't notice as I assumed it would have had some type of error when I built the image!